packages/components/card/src/card.visualroute.jsx

```diff
@@ -100,6 +100,27 @@ export const component = () => (
       </WrappedCard>
     </Spec>
 
+    <Spec label="Type - Flat, Theme - Dark, Disabled: true">
```

Comment:
The only visual difference for these new props is the `disabled` prop (lower the opacity of the card content)